src/include/duckdb/common/vector_operations/generic_executor.hpp

```diff
@@ -93,28 +116,40 @@ template <class A_TYPE, class B_TYPE>
 struct StructTypeBinary {
 	A_TYPE a_val;
 	B_TYPE b_val;
+	bool a_is_null;
```

Comment:
These fields are not necessary - `a_val` and `b_val` have their own `is_null` field (i.e. we should just use `a_val.is_null`)